src/style.css

```
@@ -1 +1,64 @@
 /* Styles go here */
+body {
+  margin: 0%;
+}
+
+a:active {
+  color: orange;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: white;
+}
+
+.logo_img {
+  display: block;
+  padding: 10px;
+  margin-left: 50px;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0%;
+  padding: 0%;
+  list-style: none;
+}
+
+.nav {
+  display: inline-block;
+}
+
+.nav__link {
+  display: inline-block;
+  position: relative;
+  text-decoration: none;
+  margin: 0 20px;
+  color: black;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  letter-spacing: 0 em;
+  text-align: left;
+  height: 60px;
+  line-height: 60px;
+}
```

```suggestion
.nav__item:not(:last-child) {
  margin-right: 20px;
}

.nav__link {
  display: inline-block;
  position: relative;
  text-decoration: none;
  color: black;
  font-family: "Roboto", sans-serif;
  font-size: 12px;
  font-weight: 500;
  text-transform: uppercase;
  padding: 23px 0;
}

.is-active {
  color: #00acdc;
}
```
set margin for .nav-item, it is bad practice to set directly height, better to make height with paddigns.